dat/missions/zalek/neburesearch/neburesearch_05.lua

```diff
@@ -0,0 +1,377 @@
+        student(fmt.f(_([["Alright, let's go to the {sys} system. I'm going to monitor the prototype to make sure everything is right."]]), {sys=testing_sys:name()}))
+        vn.done()
+        vn.run()
+        mem.stage = 2
+        misn.markerMove(mem.misn_marker, testing_sys)
+    end
+end
+
+function jumpin()
+    if mem.stage == 2 and system.cur() == testing_sys then
+        hook.timer(5.0, "arrive_at_testing_sys")
+    end
+end
+
+function arrive_at_testing_sys()
+    local osd_msg = {}
+    osd_msg[1] = fmt.f(_("Fly to the checkpoint in the {sys} system."), {sys=testing_sys:name()})
+    osd_msg[2] = fmt.f(_("Return to {pnt} in the {sys} system."), {pnt=homeworld:name(), sys=homeworld_sys:name()})
+    mem.stage = 3
+    vn.clear()
+    vn.scene()
+    local student = vn.newCharacter( nebu_research.vn_student() )
+    vn.transition("fade")
+    point_pos = vec2.new(0, 0)
+    marker = system.mrkAdd(point_pos, _("Checkpoint"))
+    vn.na(fmt.f(_("You arrived in the {sys} system and the student signals you that everything works as expected."), {sys=testing_sys:name()}))
+    student(fmt.f(_([["To proceed, please fly towards the checkpoint that I just marked on your map. Afterwards we'll return to the {sys} system."]]), {sys=dest_sys:name()}))
+    vn.done()
+    vn.run()
+    misn.osdCreate(osd_title, osd_msg)
+    hook.timer(1.0, "timer")
+end
+
+function hasShieldingPrototypeEquiped()
+    local o = player.pilot():outfits("structure")
+    for i=1,#o do
+        if o[i]:nameRaw()=="Nebular Shielding Prototype" then
+            return true
+        end
+    end
+    return false
+end
+
+function timer()
+    if player.pos():dist(point_pos) < 500 then
+        local osd_msg = {}
+        osd_msg[1] = _("Wait...")
+        osd_msg[2] = fmt.f(_("Return to {pnt} in the {sys} system."), {pnt=homeworld:name(), sys=homeworld_sys:name()})
+        mem.stage = 4
+        system.mrkRm(marker)
+        misn.osdCreate(osd_title, osd_msg)
+        beginScan()
+    else
+        hook.timer(1.0, "timer")
+    end
+end
+
+function beginScan()
+    ghost = pilot.add("Thurion Ingenuity", "Thurion", point_pos + vec2.new(600, 1200), _("Ghost Ship"))
+    ghost:outfitRm("cores")
+    ghost:outfitAdd("Nexus Light Stealth Plating")
+    ghost:outfitAdd("Milspec Thalos 3602 Core System")
+    ghost:outfitAdd("Tricon Zephyr II Engine")
+    ghost:setInvincible()
+    ghost:control()
+--    ghost:follow(player.pilot())
+    moveto()
+    hook.timer(5.0, "moveto")
+    hook.timer(20.0, "endScan")
+end
+
+function moveto()
+    if mem.stage < 5 then
+        ghost: moveto(player.pos(), false, false)
+        hook.timer(5.0, "moveto")
+    end
+end
+
+function endScan()
+    local osd_msg = {}
+    local ps = player.pilot()
+    mem.stage = 5
+    osd_msg[1] = _("Survive.")
```

Comment:
I don't think most of the OSD use punctuation at the end (no `.`, but sometimes `!`)